python/pypto/language/parser/ast_parser.py

```
@@ -1691,82 +1698,50 @@ def _resolve_list_kwarg(self, value: ast.List) -> Any:
-            return op_func(*args, **kwargs, span=call_span)
+        if hasattr(module, op_name):
+            op_func = getattr(module, op_name)
+            return op_func(*args, **kwargs, span=self.span_tracker.get_span(call))
 
         raise InvalidOperationError(
-            f"Unknown tensor operation: {op_name}",
+            f"Unknown {module_name} operation: {op_name}",
             span=self.span_tracker.get_span(call),
-            hint=f"Check if '{op_name}' is a valid tensor operation",
+            hint=f"Check if '{op_name}' is a valid {module_name} operation",
         )
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)